addons/udes_get_info/README.md

```diff
@@ -0,0 +1,15 @@
+# UDES Get Info
+Add get_info methods for udes stock based APIs
```

Comment:
Can we add more details about what is this module providing and what is it usually used for? It is not very clear just from reading this README in my opinion.

Reply:
Sure. will check and get the readme updated